packages/core/data/oot/world_mq/deku_tree_mq.yml

```diff
@@ -56,8 +104,25 @@
     DEKU_BLOCK: "true"
   locations:
     "MQ Deku Tree Scrub": "can_hit_scrub && scrub_price(0x1b)"
+    "MQ Deku Tree Grass Basement Upper 1": "can_cut_grass"
+    "MQ Deku Tree Grass Basement Upper 2": "can_cut_grass"
+    "MQ Deku Tree Grass Basement Upper 3": "can_cut_grass"
+    "MQ Deku Tree Grass Larvae Room 1": "is_child && can_cut_grass" #You can go backwards to get this specific grass with nothing else.
+    "MQ Deku Tree Grass Larvae Room 2": "is_child && can_cut_grass" #You can go backwards to get this specific grass with nothing else.
```

Comment:
put these in backrooms, not here. could go through the tunnel then age swap to adult.

Reply:
Fixed.